src/Mod/Draft/draftmake/make_label.py

```diff
@@ -280,9 +282,13 @@ def make_label(target_point=App.Vector(0, 0, 0),
     if not custom_text:
         custom_text = "Label"
     try:
-        utils.type_check([(custom_text, str)], name=_name)
+        utils.type_check([(custom_text, (str, list))])
     except TypeError:
-        _err(translate("draft","Wrong input: must be a string."))
+        _err(translate("draft","Wrong input: must be a list of strings or a single string."))
+        return None
+
+    if not all(isinstance(element, str) for element in custom_text):
```

Comment:
This seems wrong: If `custom_text` is not a list, but a string, I think this will end up looping over each element of the string to check they are strings (since elements of a string are single-character strings themselves, the check works, but this wastes processing).

Reply:
I just realized that your second commit actually introduces this problem, with just the first commit `custom_text` is always a list at this point.

Reply:
> with just the first commit `custom_text` is always a list at this point.

This cannot be correct. The 2nd commit removed code on line 294 and 295.

Reply:
Yes, so "with just the first commit" (in other words, if you omit the second commit), then those lines still exist, and those lines ensure that `custom_text` is also a list. So removing those lines means that `custom_text` might be a string instead of a list, so the because of that you might end up iterating a string instead of a list.

Right?

Reply:
No the code that ensured custom_text was a list came after line 290 and had no impact there.

Reply:
Oh, you're right, I hadn't looked properly. You're right, it was broken in the first commit already :-) Seems fixed now!